src/main/features/response/components/payment-intention/payment-option.ts

```diff
@@ -72,16 +73,17 @@ export class PaymentOptionPage {
   }
 
   private renderView (form: Form<DefendantPaymentOption>, res: express.Response) {
+    const claim: Claim = res.locals.claim
+
     function isApplicableFor (draft: ResponseDraft): boolean {
-      if (!FeatureToggles.isEnabled('statementOfMeans')) {
+      if (!FeatureToggles.hasAnyAuthorisedFeature(claim.features, 'admissions')) {
         return false
       }
-      return draft.isResponseFullyAdmitted()
+      return draft.isResponseFullyAdmitted(claim.features)
```

Comment:
I do not like the fact that now we have to pass features to draft methods. IMO we should separate logic of checking whether feature is enabled from logic checking state of the draft.

I would suggest:

```
# new helper method in separate file
areAdmissionsEnabled(claim: Claim): boolean {
  return FeatureToggles.hasAnyAuthorisedFeature(claim.features, 'admissions')
}

// in many places
ClaimFeatureToggles.areAdmissionsEnabled(claim) && draft.isResponseFullyAdmitted()

// or is other places
if (ClaimFeatureToggles.areAdmissionsEnabled(claim)) {
  if (draft.isResponseFullyAdmitted()) {
    // some logic
  }
}
```